package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wp-parsely",
-  "version": "2.4.1",
+  "version": "2.5.0-alpha",
```

Comment:
Per https://docs.npmjs.com/cli/v7/configuring-npm/package-json#version `version` field is optional if there are no plans to publish it.

Can we consider adding the [private field](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private), to stop it being published, and then remove this `version` field, so there is one less place to update the version on each release?